packages/shared/src/components/modals/RanksModal/RanksTags.tsx

```diff
@@ -3,14 +3,17 @@ import { ModalSubTitle } from '../common';
 import { RanksTagsProps, RanksTagsSection, RanksTagsList } from './common';
 import RankTagItem from './RankTagItem';
 
-const RanksTags = ({ tags }: RanksTagsProps): ReactElement => {
+const RanksTags = ({ tags, limit = 6 }: RanksTagsProps): ReactElement => {
   return (
     <RanksTagsSection>
       <ModalSubTitle>Reading status per tag</ModalSubTitle>
-      <RanksTagsList>
-        {tags.map((tag) => (
-          <RankTagItem tag={tag} key={tag.tag} />
-        ))}
+      <RanksTagsList style={{ gridTemplateRows: '1fr 1fr 1fr' }}>
```

Comment:
Why don't we use the classname for this?

Reply:
I tried `grid-rows-['1fr 1fr 1fr']` but  didn't work.

Reply:
It should be `grid-rows-3`

Reply:
`grid-rows-3` provides a different value specifically `grid-template-rows: repeat(3, minmax(0, 1fr));` - which works differently.